inc/sainexthopgroup.h

```diff
@@ -263,7 +276,8 @@ typedef enum _sai_next_hop_group_member_attr_t
      *
      * Index specifying the strict member's order.
      * Allowed value range for is from 0 to SAI_NEXT_HOP_GROUP_ATTR_REAL_SIZE - 1.
-     * Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_FINE_GRAIN_ECMP.
+     * Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_FINE_GRAIN_ECMP
```

Comment:
Should also extend SAI_NEXT_HOP_GROUP_MEMBER_ATTR_NEXT_HOP_ID to support pointing to objects of type SAI_OBJECT_TYPE_NEXT_HOP_GROUP (currently can only point to a single next hop)